Tech Journey/My-#100DayOfCode-Journey-So-Far.md

```diff
@@ -0,0 +1,56 @@
+## A bit about me, Who am I?
```

Comment:
Always add a space after heading
If you write on vscode, you can download an extension markdown linter
It will help you

Reply:
I just wrote a draft and then just worked on it, adding the space